### ISSUE TITLE
GC: Backup: Operation Test Failure

### ISSUE THREAD

**Issue by dadams39** [alcionai/corso#1743]
Upgrade of MS Graph libraries has caused certain fields and functions to be deprecated. These affect the functions used in  Incremental backup implementation. This failure is associated with the following test: `TestBackupOpIntegrationSuite/TestBackup_Run_exchange/Integration_Exchange.Mail`
```java
/home/runner/work/corso/corso/src/internal/operations/backup_test.go:412
          	Error:      	Received unexpected error:
          	            	1 error occurred:
          	            		* getting nested object handle: error reading directory: entry not found
```

**Comment by dadams39** [comment]
Stack Tracing from the above error:
- L412 `checkMetadataFilesExist()`
- L225 `kw.RestoreMultipletems()` --> This leaves the test and goes to `src/internal/kopia/wrapper.go`
- L725 `getItemStream` 
- L651 `GetNestedEntry()` --> from Kopia Upstream
- L656 is where the above error is generated.

**Comment by dadams39** [comment]
Bug fixed within [d035a37](https://github.com/alcionai/corso/commit/d035a3767908d05dcf940a0d947e31dfec607e12)
